readme.md

```diff
@@ -15,7 +15,7 @@ It achieves this in 2 ways:
 
 ### Installation
 
-> **TL;DR** Copy contents of `dist/` to `%AppData%/Local/Finkit/ManicTime/Plugins/Packages`.
+> **TL;DR** Copy contents of `dist/` to `%systemdrive%\Users\%username%\AppData\Local\Finkit\ManicTime\Plugins`.
```

Comment:
Could these be '/' instead? So that it is consistent with the rest of the document.

Reply:
No, dont be stupid! :D

Reply:
Could you change all the '/' to '\' please? To keep the document consistent?

Reply:
Fail!

Reply:
Added updated changes